### PR TITLE
TensorFlow Min, Max layer test initial implementation

### PR DESCRIPTION
**Pull request by gkrivor**
### Details:
 - Added TensorFlow Min, Max layer test initial implementation

### Tickets:
 - 92867

### CODE REVIEW

File: tests/layer_tests/tensorflow_tests/test_tf_MinMax.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# Testing Min, Max operations (Initial Implementation)
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/Min
+#                https://www.tensorflow.org/api_docs/python/tf/raw_ops/Max
+
+class TestMinMaxOps(CommonTFLayerTest):
+    # input_shape - should be an array
+    # axis - array which points on axis for the operation
+    # op_type - type of tested operation
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
```
Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_MinMax.py:18]: please remove it

File: tests/layer_tests/tensorflow_tests/test_tf_MinMax.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    def create_minmax_ops_placeholder_const_net(self, input_shape, axis, op_type, keep_dims, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->op_type   =>       Placeholder->ReduceMin/Max
+
+        """
+
```
Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_MinMax.py:26]: please remove it

File: tests/layer_tests/tensorflow_tests/test_tf_MinMax.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            op_type_to_tf = {
+                'Min': tf.raw_ops.Min,
+                'Max': tf.raw_ops.Max,
+            }
+            tf_input_shape = input_shape.copy()
```
Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_MinMax.py:37]: please do not copy a shape

File: tests/layer_tests/tensorflow_tests/test_tf_MinMax.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            tf_input = tf.compat.v1.placeholder(tf.float32, tf_input_shape, 'Input')
+            tf_axis = tf.constant(axis)
```
Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_MinMax.py:39]: please explicitly specify a type.

File: tests/layer_tests/tensorflow_tests/test_tf_MinMax.py [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+        ref_net = None
+
+        return tf_net, ref_net
+
+    test_data = [
+        pytest.param(
+            dict(input_shape=[2, 3], axis=[1]),     #Simple test
+            marks=pytest.mark.precommit_tf_fe),
+        dict(input_shape=[2, 3, 3, 4], axis=[2]),   #Simple test with possible nchw/nhwc
```
Comment (rkazants) [inline tests/layer_tests/tensorflow_tests/test_tf_MinMax.py:54]: please add negative axis case

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**